static/src/javascripts/projects/common/modules/experiments/tests/video-series-page.js

```diff
@@ -0,0 +1,36 @@
+define([
+    'common/utils/config'
+], function (
+    config
+) {
+    return function () {
+
+        this.id = 'VideoSeries';
+        this.start = '2016-04-21';
+        this.expiry = '2016-04-25';
+        this.author = 'James Gorrie';
+        this.description = 'New video series page (initial numbers test)';
+        this.audience = 1.0;
+        this.audienceOffset = 0.0;
+        this.successMeasure = '';
```

Comment:
To have them there - I can add them in when we put the actual test in (which will be using the same files)